### ISSUE TITLE
Add new example Jupyter notebooks / edit old ones

### ISSUE THREAD

**Issue by JeS24** [einsteinpy/einsteinpy#626]
### 🐞 **Problem**
We need new [example notebooks](https://github.com/einsteinpy/einsteinpy/tree/main/docs/source/examples) or edits to older notebooks to highlight the feature additions and API changes that EinsteinPy has seen over the years. Recently, two PRs added features that could use a notebook demonstrating them. This issue will track multiple features of EinsteinPy that should be properly documented.
- [ ] PR #625 added support for custom metrics functions, outside of electro-vacuum (EV) solutions. We need one or two examples showcasing this (preferably with a non-EV metric).
- [ ] PR #612 (issue #610) improved scaling in geodesic plots. Most of the related example notebook were updated (See discussion in #612), but [one](https://github.com/einsteinpy/einsteinpy/blob/main/docs/source/examples/Visualizing%20Precession%20in%20Schwarzschild%20Spacetime.ipynb) had to be left out at the time due to merge conflicts. It can be edited now.
- [ ] *More to be added*

***If you are interested in contributing towards this issue, please do not hesitate to ask for clarifications.***

### 📋  **Steps to solve the problem**
If you are a new contributor, see this: https://docs.einsteinpy.org/en/latest/dev_guide.html.

 * Comment below about what you've started working on.
 * Add, commit, push your changes.
 * Submit a pull request.
 * Add this in the comments:
 	- `Addresses #<put issue number here>` if you are partially fixing the issue.
 	- `Fixes #<put issue number here>` if you are completely fixing the issue.
 * Ask for a review in comments section of pull request.
 * Celebrate your contribution to this project 🎉.

**Comment by Snerdify** [comment]
Hey has the issue already assigned or can i take it up? @JeS24

**Comment by vigneshRamanathan3105** [comment]
Is this issue still open?

**Comment by JeS24** [comment]
@vigneshRamanathan3105 Yes.

**Comment by vigneshRamanathan3105** [comment]
Is this free to pick up?

**Comment by JeS24** [comment]
@vigneshRamanathan3105 Yeah, it is. This issue will have multiple parts and multiple PRs. So, just make sure to comment here what you've started working on, so that others are in the loop. For example, if you work on an example notebook wrt #625, you can mention that here.

**Comment by thisisdilmurod** [comment]
Good day! It's my first try. Just wanted to ask if this issue is still valid.

**Comment by JeS24** [comment]
> Good day! It's my first try. Just wanted to ask if this issue is still valid.

Good day. This issue is open for contributions.

**Comment by jjmcinto** [comment]
I updated some plots in _Visualizing Precession in Schwarzschild Spacetime.ipynb_ to demonstrate title and colour selection functionality.

I will attempt to send a pull request shortly.

**Comment by jjmcinto** [comment]
Nope, that did not work. I need authorization to push:

> git commit -m "Addresses #610"
[main 4209569] Addresses #610
 2 files changed, 98949 insertions(+), 98932 deletions(-)
 rewrite docs/source/examples/Visualizing Precession in Schwarzschild Spacetime.ipynb (71%)

> git push
remote: Permission to einsteinpy/einsteinpy.git denied to jjmcinto.
fatal: unable to access 'https://github.com/einsteinpy/einsteinpy.git/': The requested URL returned error: 403

So, who can I talk to about adding my changes to the documentation?

**Comment by JeS24** [comment]
@jjmcinto You need to [fork this repo](https://github.com/einsteinpy/einsteinpy/fork) first. Then, commit & push changes to that fork and open a PR. See: https://docs.github.com/en/get-started/quickstart/contributing-to-projects or https://docs.einsteinpy.org/en/latest/dev_guide.html for EinsteinPy-specific development details.

**Comment by aSimpson02** [comment]
Hi I would like to have a go at the first issue if thats still available?

**Comment by JeS24** [comment]
@aSimpson02 Hi. That issue's available. Feel free to contribute.

**Comment by mushtaq96** [comment]
Hi @aSimpson02 , did you already finish working on this task?
